spatial-history/index.md

```diff
@@ -30,10 +13,23 @@ It makes sense to put an introductory paragraph here, above the map and buttons.
+
+## **Paratroopers**
+
+If the paratroopers landed off target (which they did on D-Day), they would need to be able to orientate themselves without adequate knowledge of the ground below. This was an incredible accomplishment considering that France had been occupied by Germany since 1940 making the only source of detailed maps of the area arial photos or from resistance groups. For example, Sword and Utah beach were both key targets that involved the airborne divisions before the invasion began [Jesse Greenspan](https://www.history.com/news/landing-at-normandy-the-5-beaches-of-d-day). At these locations, paratroopers missed their marks and landings but thanks to proper training and detailed mappings of the area, not only were they able to destroy their key objectives, but they were also able to link up with the invading forces once they landed on the beaches as well and provide support [Ibid](https://www.history.com/news/landing-at-normandy-the-5-beaches-of-d-day). It is noteworthy to consider how the paratroopers and invading forces formed a pincer like movement in two major ways that is actually very intuitive. Firstly, Utah and Sword were the two most extreme easterly and westerly beaches (See the Map above) meaning that as they were taken, they could apply pressure to the sectors in the middle forcing the Nazis to have to worry about forces moving across. Second, was the paratroopers being able to link up with the landing forces meant that they attacked from behind the defending Germans while their fellow soldiers attacked from the front. This would have forced the Nazi’s to fight on multiple fronts.
+
+## **The Beaches Themselves**
+
```

Comment:
- Careful of minor typos (apostrophes ,etc) here & throughout. 
- that are some OLD SCHOOL attributes on that image. Use CSS instead, in a `style="attr: value;"` construct